public/js/ticTacToe.js

```diff
@@ -0,0 +1,112 @@
+var tiles = document.getElementsByClassName("tile");
+var state = [0, 0, 0, 0, 0, 0, 0, 0, 0];
+var game = true;
+var visible = false;
+var HUMAN = false;
+var COMPUTER = true;
+var HUMVAL = -1;
+var COMVAL = 1;
+var currentPlayer = HUMAN;
+var winMatrix = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6]
+];
+
+function reset() {
+    for (var x = 0; x < 9; x++) {
```

Comment:
9? My detective skills tell me it may represent the number of tiles on the game board. Something that was suggested to me several times is to define constants to make these  things clear.

const MAX_TILES = 9
for ( var x = 0; x < MAX_TILES; x++ )